parastell/nwl_utils.py

```diff
@@ -186,6 +186,9 @@ def flux_coords(plas_eq, wall_s, coords, num_threads):
             for theta_coord_chunk in theta_coord_chunks
             for theta_coord in theta_coord_chunk
         ]
+    # Ensures theta_coords are all positive (add 360 degrees where needed)
+    theta_coords = (theta_coords + 2 * np.pi) % 2*np.pi
```

Comment:
I think this is why black is complaining.
```suggestion
    theta_coords = (theta_coords + 2 * np.pi) % 2 * np.pi
```

Reply:
Thanks!